src/applications/vaos/newAppointmentFlow.js

```diff
@@ -90,7 +103,14 @@ export default {
     url: '/new-appointment/va-facility',
     next: 'reasonForAppointment',
     // TODO: If user is not CC eligible, return to page prior to typeOfFacility
-    previous: 'typeOfFacility',
+    previous(state) {
+      let nextState = 'typeOfCare';
+
+      if (isCommunityCare(state)) {
```

Comment:
I don't think checking this is enough to tell us that they should go back to the type of facility page, since this check only tells us that they picked a type of care that can be used for CC. 

I think if you also checked that `getFormData(state).facilityType` wasn't undefined, that would be enough. If there's something in that `facilityType` field, it means they've been on the `typeOfFacility` page.

Reply:
Done.